src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -332,23 +335,22 @@ def execute_link(
         add_red_edge_ndvi_band: bool = False,
         add_ndwi_band: bool = False,
         cloud_detection_warning: bool = False,
-        fill_with_nearest_date: bool = False,
         fill_coordinates: [] = [],
     ):
         """
         Executes the download, crops and the calculates the NVDI for a specific link.
 
         @param link: Link to a file from the NSO.
-        @param delete_zip_file: whether or not to keep the original .zip file on default it will keep the file.
-        @param delete_source_files: whether or not to keep the extracted files on defualt it will delete the source files.
+        @param delete_zip_file: whether or not to keep the original .zip file on default it will keep the file, defaults to keeping the zip file. For unnecessary redownloading a .zip file
+        @param delete_source_files: whether or not to keep the extracted files on default it will delete the source files.
```

Comment:
I would write "the extracted files. By default it will", to be clearer.

Reply:
fixed now